src/api/deriveBundleData/graph/processReasons.ts

```diff
@@ -43,6 +43,6 @@ export function processReasons(reasons: Reason[], moduleIdToNameMap: ModuleIdToN
         parents: [...directParents, ...lazyParents],
         directParents: [...directParents],
         lazyParents: [...lazyParents],
-        entryType,
+        ...(entryType ? { entryType } : {}),
```

Comment:
>...(entryType ? { entryType } : {}), [](http://example.com/codeflow?start=8&length=36)

What does this buy us?  If it's `undefined`, I believe it gets ignored when serialized anyway.

Reply:
Huh. Learn something new.